extensions/wikia/Chat2/ChatHooks.class.php

```diff
@@ -202,4 +202,19 @@ public static function onUserGetRights( $user, &$aRights ) {
 
 		return true;
 	}
+
+	/**
+	 * Hook: BeforePageDisplay
+	 * SUS-936: Abort BeforePageDisplay hook on Special:Chat so that extensions can't add redundant assets
+	 * @param OutputPage $out
+	 * @param Skin $skin
+	 * @return bool false to stop hook processing if we're on Special:Chat
+	 */
+	public static function abortBeforePageDisplayOnSpecialChat( OutputPage $out, Skin $skin ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "$skin". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)